packages/webapp/components/search/SearchV1Page.tsx

```diff
@@ -1,29 +1,45 @@
 import React, { ReactElement } from 'react';
-import { NextSeo } from 'next-seo';
+import { NextSeo, NextSeoProps } from 'next-seo';
 import {
   SearchResult,
   SearchSourceList,
 } from '@dailydotdev/shared/src/components/search';
 import { useChat } from '@dailydotdev/shared/src/hooks';
 import { SearchContainer } from '@dailydotdev/shared/src/components/search/SearchContainer';
 import { useRouter } from 'next/router';
+import { cloudinary } from '@dailydotdev/shared/src/lib/image';
+import { labels } from '@dailydotdev/shared/src/lib';
 import { getLayout as getMainLayout } from '../layouts/MainLayout';
+import { getTemplatedTitle } from '../layouts/utils';
 
 const SearchPage = (): ReactElement => {
   const router = useRouter();
+  const query = router?.query?.q as string;
   const { data, queryKey, handleSubmit, isLoading } = useChat({
     id: router?.query?.id as string,
-    query: router?.query?.q as string,
+    query,
   });
   const content = data?.chunks?.[0]?.response || '';
 
+  const seo: NextSeoProps = {
+    title: getTemplatedTitle(data?.chunks[0]?.prompt || query || 'Search'),
+    description: content || labels.search.shortDescription,
```

Comment:
I would still splice the content very basic splice should do.
Just in case it streams a way to long answer